sql/core/src/main/scala/org/apache/spark/sql/jdbc/DB2Dialect.scala

```diff
@@ -49,4 +49,16 @@ private object DB2Dialect extends JdbcDialect {
   }
 
   override def isCascadingTruncateTable(): Option[Boolean] = Some(false)
+
+  /**
+  * The SQL query used to truncate a table.
+  * @param table The JDBCOptions.
+  * @param cascade Whether or not to cascade the truncation. Default value is the
+  *                value of isCascadingTruncateTable()
```

Comment:
For dialects not supporting cascading, should the documentation perhaps indicate that the `cascade` parameter is ignored?